content/workshops/_index.md

```diff
@@ -20,54 +23,12 @@ Overall page structure and ideas to convey
 - Build a platform to make collaboration easier between teams
 
 -->
-## Introduction
 {{<break 1>}}
-Generally speaking, building an online application is not easy. Having to manage deployment and resources can take a big margin of your development power and slow down production. Cdev is created with production and easy deployment in mind. Once you have your development ambient setup and your program coded, it only takes one line of code and a few seconds to have your application up and running on AWS.
-
-{{<break 1>}}
-
-## Goals of the Workshop
-
+# Workshops
 {{<break 1>}}
+Generally speaking, building an online application is not easy. Having to manage deployment and resources can take a lot of time and energy, slowing down production. Cdev is created with production and simple deployments in mind. Once you have your program written, it only takes one command in the terminal, and a small amount of time while you wait for the deployment to finish, and your program has a live deployment on the internet! 
 
-We have the following goals:
-- Create an API together.
-- Setup routes.
-- Setup Auth0 and have authentication working.
-- Test your API using Insomnia.
-- Use your API with a frontend template.
+We have set up several different workshops to help get you started using Cdev and achieve your programming goals. Check out the available options below, and also feel free to contact us to set up a personalized workshop for your school or organization.  
```

Comment:
Lets change the end of this to be a stronger Call To Action (CTA) with information about signing up for a workshop; similair to the CTA at the end of the other pages.